<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>PID</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item alignment="Qt::AlignBottom">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Angle</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Pitch</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item alignment="Qt::AlignRight">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>P</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="angle_pitch_p_box">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="decimals">
                   <number>2</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item alignment="Qt::AlignRight">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>I</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="angle_pitch_i_box">
                  <property name="decimals">
                   <number>2</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item alignment="Qt::AlignRight">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>D</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="angle_pitch_d_box">
                  <property name="decimals">
                   <number>2</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Roll</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item alignment="Qt::AlignRight">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>P</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="angle_roll_p_box">
                  <property name="decimals">
                   <number>2</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item alignment="Qt::AlignRight">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>I</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="angle_roll_i_box">
                  <property name="decimals">
                   <number>2</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item alignment="Qt::AlignRight">
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>D</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="angle_roll_d_box">
                  <property name="decimals">
                   <number>2</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Yaw</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item alignment="Qt::AlignRight">
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>P</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="angle_yaw_p_box">
                  <property name="decimals">
                   <number>2</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item alignment="Qt::AlignRight">
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>I</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="angle_yaw_i_box">
                  <property name="decimals">
                   <number>2</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <item alignment="Qt::AlignRight">
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>D</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="angle_yaw_d_box">
                  <property name="decimals">
                   <number>2</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item alignment="Qt::AlignBottom">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Rate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_20">
                <property name="text">
                 <string>Pitch</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_16">
                <item alignment="Qt::AlignRight">
                 <widget class="QLabel" name="label_21">
                  <property name="text">
                   <string>P</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="rate_pitch_p_box">
                  <property name="decimals">
                   <number>2</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_17">
                <item alignment="Qt::AlignRight">
                 <widget class="QLabel" name="label_22">
                  <property name="text">
                   <string>I</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="rate_pitch_i_box">
                  <property name="decimals">
                   <number>2</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_18">
                <item alignment="Qt::AlignRight">
                 <widget class="QLabel" name="label_23">
                  <property name="text">
                   <string>D</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="rate_pitch_d_box">
                  <property name="decimals">
                   <number>2</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_24">
                <property name="text">
                 <string>Roll</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_19">
                <item alignment="Qt::AlignRight">
                 <widget class="QLabel" name="label_25">
                  <property name="text">
                   <string>P</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="rate_roll_p_box">
                  <property name="decimals">
                   <number>2</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_20">
                <item alignment="Qt::AlignRight">
                 <widget class="QLabel" name="label_26">
                  <property name="text">
                   <string>I</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="rate_roll_i_box">
                  <property name="decimals">
                   <number>2</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_21">
                <item alignment="Qt::AlignRight">
                 <widget class="QLabel" name="label_27">
                  <property name="text">
                   <string>D</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="rate_roll_d_box">
                  <property name="decimals">
                   <number>2</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_28">
                <property name="text">
                 <string>Yaw</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_22">
                <item alignment="Qt::AlignRight">
                 <widget class="QLabel" name="label_29">
                  <property name="text">
                   <string>P</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="rate_roll_p_box_2">
                  <property name="decimals">
                   <number>2</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_23">
                <item alignment="Qt::AlignRight">
                 <widget class="QLabel" name="label_30">
                  <property name="text">
                   <string>I</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="rate_roll_i_box_2">
                  <property name="decimals">
                   <number>2</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_24">
                <item alignment="Qt::AlignRight">
                 <widget class="QLabel" name="label_31">
                  <property name="text">
                   <string>D</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="rate_roll_d_box_2">
                  <property name="decimals">
                   <number>2</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Other</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_12"/>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="read_button">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Read</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="send_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>622</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
